cmd/cloud-controller-manager/app/options/options_test.go

```diff
@@ -35,7 +38,6 @@ func TestAddFlags(t *testing.T) {
 	s.AddFlags(f)
 
 	args := []string{
-		"--address=192.168.4.10",
```

Comment:
why are those removed?

Reply:
The Address is set in `InsecureServingOptions.ApplyTo`, so removed here as kube-apiserever options test.